content/tutorial/tutorial.md

```diff
@@ -1112,7 +1112,7 @@ Game komponentinin `render` funskiyasında, açarı `<li key={move}>` formada ya
 
 **[Bu nöqtəyə kimi olan bütün kod](https://codepen.io/gaearon/pen/PmmXRE?editors=0010)**
 
-`jumpTo` funskiyasın olmadığından siyahıdakı hər hansı bir düyməni tıkladıqda xəta baş verir. `jumpTo` funskiyasını tətbiq etməmişdən qabaq, biz hansı gedişdə olduğumuzu bilmək üçün `stepNumber` state-ini Game komponentinə əlavə edəcəyik.
+`jumpTo` funksiyasın olmadığından siyahıdakı hər hansı bir düyməni tıkladıqda xəta baş verir. `jumpTo` funksiyasını tətbiq etməmişdən qabaq, biz hansı gedişdə olduğumuzu bilmək üçün `stepNumber` state-ini Game komponentinə əlavə edəcəyik.
```

Comment:
I just noticed my own translation mistake here. Since you are already working on it, could you plese change "funksiyasın" to "funksiyasının"?